cypress/support/Utils/employee-utils.ts

```diff
@@ -35,3 +37,28 @@ export function writeEmployeeReport(jobTitle: string, locationName: string) {
     selectionCriteriaData: [jobTitle, locationName],
   });
 }
+
+export function writeEventFixture() {
+  cy.writeFile("cypress/fixtures/event.json", {
+    name: `test${Math.floor(Math.random() * 100)}`,
+    description: "",
+    status: true,
+  });
+}
+
+export function writeClaimFixture(eventId: number, amount: number) {
+  cy.writeFile("cypress/fixtures/claim.json", {
+    claimEventId: eventId,
+    currencyId: "AFN",
+    remarks: null,
+  });
+}
+
+export function writeExpenseFixture(expenseId){
+  cy.writeFile("cypress/fixtures/expense.json", {
+    expenseTypeId: expenseId,
+    amount: 100,
+    date: moment().format('YYYY-MM-DD'),
```

Comment:
please define the date functions in generic functions class, so if you have a change in the format or something else, you will have a change in one place instead of changing in all utils